recipes/pylammpsmpi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pylammpsmpi" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 78373d693b3856feff22d9cf6c86f617fe80371f9fe5ef44e433f4ebb5a9b443
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win | py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - mpi4py
+    - openmpi
```

Comment:
Please see the documentation here for MPI: https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi

Reply:
But this only works with open mpi, because mpich does not support communicating via stdin.

Reply:
Then only inlcude openmpi in `conda_build_config.yaml`

Reply:
Done